tech-audit.js

```diff
@@ -268,6 +269,10 @@
     clientSideListAdd("Twitter Widgets");
   }
 
+  if (elementExists('[data-reactroot]') || elementExists('[data-reactid]')) {
```

Comment:
This could be simplified to a single call! QuerySelector can have multiple comma-separated query segments.